content/issues/trying-out-the-new-projects-experience/about-projects.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: About projects (beta)
-intro: 'Projects are a customizable, flexible tool for planning and tracking work on {% data variables.product.company_short %}.'
```

Comment:
tools

Reply:
I'm sorry, it was my mistake! The sentence was correct.